tock/tock/utils.py

```diff
@@ -40,21 +41,56 @@ def has_permission(self, request, view):
 
 def get_float_data(endpoint, params=None):
     """Fetch Float data from given endpoint with given params. Different request
-      variables used for testing / shell work with the fake Float API
-     (see tock.mock_api_server) versus all other uses."""
-    testing = 'test' in sys.argv
-    shell = 'shell' in sys.argv
-    if testing or shell:
-        url = '{}/{}'.format(test.FLOAT_API_URL_BASE, endpoint)
+      variables used for testing / shell work with the mock Float API."""
+
+    url = '{}/{}'.format(base.FLOAT_API_URL_BASE, endpoint)
+
+    # If testing, get mock response.
+    if 'test' in sys.argv or 'shell' in sys.argv:
```

Comment:
Would it be useful to control this via an environment variable?  That way devs could also use it during manual testing, if they wanted... Anyways, just an idea.

I am a *little* bit wary of doing a test like `if 'somestring' in sys.argv` just because of the possibility that the string might appear somewhere in some other use case and trigger behavior that no one expects--I'd feel a bit more confident in this if it e.g. checked if `manage.py` was in it?  I've typically done something like [this](https://github.com/toolness/hive-django/blob/master/hive/settings_utils.py#L45-L47):

```py
def is_running_test_suite():
    return (os.path.basename(sys.argv[0]) == 'manage.py' and 
            len(sys.argv) > 1 and sys.argv[1] == 'test')
```